ext/sodium/libsodium.stub.php

```diff
@@ -58,6 +58,34 @@ function sodium_crypto_box_seal_open(string $ciphertext, string $key_pair): stri
 
 function sodium_crypto_box_secretkey(string $key_pair): string {}
 
+#ifdef crypto_core_ristretto255_HASHBYTES
+function sodium_crypto_core_ristretto255_add(string $p, string $q): string {}
+
+function sodium_crypto_core_ristretto255_from_hash(string $s): string {}
+
+function sodium_crypto_core_ristretto255_is_valid_point(string $s): bool {}
+
+function sodium_crypto_core_ristretto255_random(): string {}
+
+function sodium_crypto_core_ristretto255_scalar_add(string $p, string $q): string {}
```

Comment:
The documentation at https://doc.libsodium.org/advanced/point-arithmetic/ristretto seems to use `$x`, `$y` for the scalar APIs and `$p`, `$q` for the non-scalar ones.

Reply:
Good catch, I will correct this.